feature-libs/storefinder/components/store-finder-store-description/store-finder-store-description.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="container">
     <div class="row">
       <article class="cx-store col-md-4">
-        <h2>{{ location.displayName || location.name }}</h2>
+        <h2 role="alert">{{ location.displayName || location.name }}</h2>
```

Comment:
@Matejk00 I assume you have tried using aria-live="assertive" but it did not work on JAWS/NVDA? Because role is very intrusive

Reply:
Yes, I tried to use `aria-live="assertive"` solution, but had an issue on JAWS. The problem was that title has been not vocalised. 

With `role="alert"` approach vocalising is working on each screen reader.

Reply:
Thanks, I tried to use the solution you proposed, but still JAWS cannot read the content in "hidden" div container.
Is it working for you?

Also another thing to consider is: do we need to create special markup in components for a11y purpose only? I noticed that in storefinder we adjust this approach in some areas. Of course if it's working better than `role="alert"` it's totally fine (because no additional alert prefix during vocalising), but on the other hand it's more code to maintenance in the future and to be honest `cx-visually-hidden` styling is little bit "hacky" :)

Reply:
> Yes the code I proposed does work for me but on Voice Over.

@BenjaminEmiliani would you mind to test your solution (https://github.com/SAP/spartacus/pull/14425#discussion_r758555957) on JAWS? For me is not working even if I put this near to the top of component's markup.

Not sure about requirements here, if it should work only on Voice Over that's fine, but I thought that we need to cover all Screen Readers: Voice Over, JAWS, NVDA. So for compatibility `role="alert"` approach works better imo.

Reply:
@Matejk00 My issue is that I am only able to test screen readers on VO as I only own a Mac machine. And yes, I do believe that all Screen Reader implementations should be compatible on Screen Reader/machine.

Reply:
When you tried the solution on JAWS, did you add the hidden div element outside the <ng-container>?

Reply:
Yes, I'm trying it outside the `<ng-container>` and with VO works fine, but not working on JAWS and NVDA.
I don't have any other ideas how to resolve this issue to work properly with all Screen Readers, so should we stay with `role="alert"`solution?

Let's also add @developpeurweb into the loop.

Reply:
@Matejk00 @BenjaminEmiliani 
- **Solution 1:** More and more I'm convinced the issue for the whole Find a Store section is **focus management**, it's less evident in VO, but very clear in JAWS/NVDA, because on every UI change the **skipper gets vocalized**, so the first tab on every subpage gets the user to the skipper instead of wherever they were. In this case, it would be great to have the focus reset and the `<title>` to update as the user moves along. E.g.: 
  - Find a Store
  - Find a Store - 24 results found
  - Find a Store - Poland, 2 results found
  - Find a Store - SAP Labs Polska
- **Solution 2:** If the above is not possible, then I see `aria-live` not working mainly because `cx-store-finder-grid` and `cx-store-finder-stores-count` are not preloaded, they get created and destroyed on demand. Not even the `aria-live` number of results get vocalized on JAWS/NVDA. So, as Ben said, a dedicated service preloaded and constantly present, at the top or the bottom of the page will help vocalize messages we can't push otherwise. Same as how `cx-global-message` is preloaded, this one works for all SR.
- **Solution 3:** there are tickets addressing similar and previous issues, we could **park this ticket** until we know if any of these tickets provide a solution.
  - GH-13556
  - GH-13539
  - GH-14371
- **Solution 4:** if any of the aforementioned fixes are **NOT possible**. Then, reluctantly, we will have to use `role="alert"` to tell users they have moved to a different page.